client/blocks/importer/wordpress/import-everything/pre-migration/index.tsx

```diff
@@ -270,6 +273,10 @@ export const PreMigrationScreen: React.FunctionComponent< PreMigrationProps > =
 	}
 
 	function renderUpdatePluginInfo() {
+		dispatch(
+			recordTracksEvent( 'calypso_site_migration_install_jetpack_screen', migrationTrackingProps )
+		);
+
```

Comment:
We probably don't want this here because this triggers on every render, we already have the event in the `UpdatePluginInfo` component, maybe we can reuse `calypso_site_importer_show_update_info` event here? Also, when user is on this screen, they won't have `source_site_id` because the site is not connected yet.

Reply:
You are right, @ouikhuan. Thanks for the review and the nice feedback!!!
I could have tested the previous approach better, and there were multiple triggers, which was wrong.

I have fixed it, and now it should work as expected.

Also, I think this component is mature for refactoring and simplifying the codebase.